client/src/app/shared/guards/wrangler-or-owner.guard.ts

```diff
@@ -29,8 +32,8 @@ export class WranglerOrOwnerGuard implements CanActivate {
     }
 
     return ( getProject ?
-      this.isWranglerOrOwner(this.ingestService.getUserAccount(), getProject) :
-      this.isWrangler(this.ingestService.getUserAccount())
+      this.authService.isWranglerOrOwner(this.ingestService.getUserAccount(), getProject) :
+      this.authService.isWrangler(this.ingestService.getUserAccount())
```

Comment:
Suggestion: Could have the authService use the ingest service and cache the account there? Use the auth service to get the account. Then use account to know if it's a wrangler or owner of a project.

Reply:
That's true. Can we do that in [#225](https://app.zenhub.com/workspaces/dcp-ingest-product-development-5f71ca62a3cb47326bdc1b5c/issues/ebi-ait/dcp-ingest-central/225)?

Reply:
Yup!